README.md

```diff
@@ -1,4 +1,61 @@
 # Filecoin Shipyard
-Filecoin Shipyard is a venue for the community to pursue and collaborate on research experiments, products, code libraries, and more. It is where innovation in userland happens, where we explore new products, tools, and services to grow the Filecoin Ecosystem.
 
-To propose or discuss a new Shipyard project, please create an [issue](https://github.com/filecoin-shipyard/shipyard/issues/new) in this repo.
+The Filecoin Shipyard is a venue for the community to collaborate on code libraries, research experiments, products and more. It's an area for users from all walks to contribute their talents to create tools that continue to keep the Filecoin ecosystem robust and thriving. 
```

Comment:
The phrase "from all walks" feels odd to me as a native English speaker without the "of life" attached. (I asked a friend and they had the same gut reaction.) However, I'd actually recommend picking a different phrase because I assume non-native English speakers wouldn't intuitively understand this expression. Maybe "of all backgrounds"?

Reply:
Changed :)